data/coalition/coalition weapons.txt

```diff
@@ -402,3 +402,41 @@ effect "ion rain impact"
 	"random spin" 2
 	"random velocity" 10
 	"velocity scale" -.1
+
+
+
+outfit "MCS Extractor"
+	category "Guns"
+	licenses
+		Coalition
+	cost 106000
+	thumbnail "outfit/mcs extractor"
+	"mass" 18
+	"outfit space" -18
+	"weapon capacity" -18
+	"gun ports" -1
+	weapon
+		sound "drill"
+		"hit effect" "drilling"
+		"velocity" 10
+		"lifetime" 1
+		"reload" 1
+		"firing energy" .78
+		"firing heat" .68
+		"shield damage" .2
+		"hull damage" 1.4
+		"minable damage" 8.2
+	description `Not legally a weapon under Heliarch definitions, the "Mineral Collection and Sorting" extraction unit consists of a set of sharp, durable blades designed to quickly break down mineable asteroids.`
+	description `	Especially popular with novice pilots looking to pay off their ships, the extractor has been a reliable tool of the Coalition mining business for millennia.`
+
+effect "drilling"
+	sprite effect/smoke
+		"no repeat"
+		scale 0.5
+	sound "explosion tiny"
+	lifetime 12
+	"velocity scale" -0.05
+	"random angle" 360
+	"random velocity" 2
+	"random spin" 2
+	"random frame rate" 2
```

Comment:
Given that this is a civilian outfit with little to no offensive capabilities, would this be better under the generic outfits file as this isn't a weapon per se? Just a question over whether or not it should be moved but it's fine leaving it in here if wished.

Reply:
It's still a weapon to the game (`Gun`, to be specific), so I felt it should be with the other weapons.
Also, while it is pretty terrible at it, it **can** be used for combat, similar to how in other games we have some joke-weapons, like the Ladle in Dark Souls 2.